tests/unit/array/mixins/test_content.py

```diff
@@ -40,6 +40,9 @@ def test_content_empty_getter_return_none(cls, content_attr, start_storage):
         da = cls()
     assert getattr(da, content_attr) is None
 
+    if cls == DocumentArrayAnnlite:
```

Comment:
for example here, DA will create an annlite instance with temp file name. I don't think anything would access the same temp file concurrently, why do we need to release the lock ?

Reply:
This is because we need to test with different parameters, it's like a for loop, need to close it before another iteration.